Gruntfile.js

```diff
@@ -346,7 +349,7 @@ module.exports = function(grunt) {
       },
       options: {
         watchTask: true,
-        proxy: "alps3.test",
+        proxy: "localhost:8080",
```

Comment:
why the change here?

Reply:
The built in server runs in ‘localhost:8080’ and this avoids the need of having to configure a host.

Reply:
Let's remove proxy at all. And use the server from browserSync package.

```
options: {
  watchTask: true,
  server: {
    baseDir: './public'
  },
  startPath: "/v3"
}
```